### DISCUSSION TITLE
there is option hide title bar (minimize, close)?

### DISCUSSION THREAD

**Discussion by Alexnoj** [hluk/CopyQ#2570] [General]
take a look to file i upload. there is option hide this section and make window more clean?

![image](https://github.com/hluk/CopyQ/assets/30004592/23c3e096-2ee5-4fa3-8c68-ddd17b75f925)

**Comment by forsakenjvg** [comment]
up, also looking for it

**Comment by hluk** [comment]
This will be possible in the next release.

You can install [the recent development build](https://ci.appveyor.com/project/hluk/copyq/builds/51849662/artifacts). To disable window title in the new version:
1. Open Action dialog (F5 shortcut).
2. Enter command: `copyq config frameless_window true`
3. Click OK dialog button to run the command.

**Comment by graphixillusion** [comment]
How to move the window around when frameless_window = true?

**Reply by hluk** [reply]
Not sure. There could be a shortcut on Windows to do that.

On many window managers on Linux, it is possible to move a window by holding Win or Alt key and dragging the window with mouse holding some specific mouse button (other mouse button usually resizes the window).

**Reply by graphixillusion** [reply]
On Windows, the alt-space shortcut make the window's menu appear, but when you choose the move option from it, the cursor changes but the window still doesn't move around...
